docs/topics/proc_deploying-the-crud-booster-using-the-oc-cli-client.adoc

```diff
@@ -31,6 +31,9 @@ $ oc new-project {project-name}
 . Navigate to the root directory of your booster.
 
 . Deploy the PostgreSQL database to OpenShift.
+Ensure that you use the following values for user name, password, and database name when creating your database application.
+The booster application is pre-configured to use this values.
+Using different values might prevent your booster application from integrating with the database.
```

Comment:
Might is IMHO wrong. It's not a possibility, it's certain.

Reply:
Let me correct that. 
I'll also add information about the yAML files that contain the pre-configured username, password and db name